### DISCUSSION TITLE
Nano

### DISCUSSION THREAD

**Discussion by Bubba-Bubba** [sxyazi/yazi#706] [1. Q&A]
Yazi is very nice. However, the default editor is nano.

I created a .config/yazi/yazi.toml file, and inserted:

1  [opener]                                                                                                                                                                       
    1  edit = [                                                                                                                                                                       
    2    { exec = '${EDITOR:=helix} "$@"', desc = "$EDITOR", block = true, for = "unix" },                                                                                            
    3    { exec = 'code "%*"',          orphan = true, for = "windows" },                                                                                                             
    4  ]                                                                                                                                                                              
    
But to no avail.
I have tried to set the config in My Nixos Home-Manager, but to no avail either. 
The boilerplate code in MyNixos, is not accepted by the system build.

https://mynixos.com/home-manager/option/programs.yazi.settings

**Comment by lpnh** [comment]
I could be mistaken, but it seems that the issue isn't with Yazi itself. It appears you haven't set the `$EDITOR` variable for your environment. To use Helix as your default editor, I recommend configuring it accordingly, or you can directly set it in your yazi.toml file like this:

```
[opener]
edit = [
	{ exec = 'hx "$@"', block = true,  for = "unix" },
	{ exec = 'code "%*"',    orphan = true, for = "windows" },
]
```

Note: I'm assuming `hx` is the binary for Helix, though I'm not certain.

**Reply by C4E5G3** [reply]
That worked, thank you Felipe
